src/main/java/com/smartling/connector/hubspot/sdk/v3/email/EmailDetail.java

```diff
@@ -0,0 +1,25 @@
+package com.smartling.connector.hubspot.sdk.v3.email;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Date;
+
+@Data
+@NoArgsConstructor
+public class EmailDetail
+{
+    public static final String INCLUDED_PROPERTIES = "includedProperties=state&includedProperties=updatedAte";
```

Comment:
Is it a typo? see my change
```suggestion
    public static final String INCLUDED_PROPERTIES = "includedProperties=state&includedProperties=updatedAt";
```